controller.py

```diff
@@ -355,10 +365,13 @@ def updates_feed(self, collection_details):
                 entries.append(entry)
 
         title = "%s Collection Updates for %s" % (collection.protocol, client.url)
-
         url_params = dict()
         if last_update_time:
-            url_params = dict(last_update_time=last_update_time)
```

Comment:
This was the bad line. putting the `datetime` object into `url_for` without converting it to a string uses the default string format, which looks like TIMESTAMP_FORMAT but is missing the 'T' and the 'Z'.